README.md

```diff
@@ -1,27 +1,64 @@
-###############
-pypodcastparser
-###############
+
+# pyPodcastParser
+
 
 pypodcastparser is a podcast parser. It should parse any RSS file, but it specializes in parsing podcast rss feeds. pypodcastparser is agnostic about the method you use to get a podcast RSS feed. Most users will be comfortable with the Requests library.
 
 
 .. _Requests: http://docs.python-requests.org/en/latest/
 
-************
-Installation
-************
+
+## Installation
+Before you start:
+
+- Services should always be run inside of virtualenvs_.
+- Dependencies are managed with pip_ and pip-tools_. Virtualenvs come with
+  pip preinstalled; pip-tools should be installed manually with ``python -m pip
+  install pip-tools``.
+- Make sure you have follow the steps
+[here](https://github.com/iheartradio/content-platform-documentation/blob/master/private_python_modules/README.md)
+so you can download our private repositories.
+
+Before the first run of the service::
+
+    $ pip install pip-tools
+    $ pip install -r requirements-dev.txt
+    $ pre-commit install
+    $ pip-compile requirements-public.in
```

Comment:
we don't have requirements-public and requirements-private files this needs to update with requirements.in and requirements.txt

Reply:
done